app/views/layouts/application.html.erb

```diff
@@ -33,6 +33,11 @@
             </span>
           </a>
         </div>
+        <div class="sign-in">
+          <% if user_signed_in? %>
+            <%= link_to 'Sign out', destroy_user_session_path, confirm: 'Sign Out', method: :delete, class: "govuk-button govuk-button--warning", style: "float: right" %>
```

Comment:
never use inline styles (`style: "float: right"`) please

Reply:
Switched to standard design system header and navigation.